README.md

```diff
@@ -485,6 +485,12 @@ Depending on your system, you might need to run `pip3`, possibly with the `-H` f
 
     $ sudo -H pip3 install gitsome
 
+For most linux users, `pip3` can be installed on your system using the `python3-pip` package. 
+
+For example, Ubuntu users can use:
+
+    $ sudo apt-get install python-pip3
```

Comment:
Note this typo was fixed in https://github.com/donnemartin/gitsome/pull/30/files.

Should be:

$ sudo apt-get install python3-pip